Source/ASEditableTextNode.mm

```diff
@@ -842,4 +842,24 @@ - (void)_delegateDidFinishEditing
     [_delegate editableTextNodeDidFinishEditing:self];
 }
 
+- (NSInteger)accessibilityElementCount
+{
+  return 1;
+}
+
+- (NSArray *)accessibilityElements
```

Comment:
We have to handle if the `self.textView` is nil. This can happen if the node was not loaded yet. We should decide if we would like to load the node at that time what should load the textView, but currently this could crash.

Reply:
@nguyenhuy @Adlai-Holler  Any thoughts on if we should force loading the view in this methods or just don't return any accessibility elements like above?

Reply:
Let's go a different route, Imho the accessibility elements should not be requested if the node is not loaded yet:

```
#pragma mark - UIAccessibilityContainer

- (NSInteger)accessibilityElementCount
{
  if (!self.isNodeLoaded) {
    ASDisplayNodeFailAssert(@"Cannot access accessibilityElementCount since ASEditableTextNode is not loaded");
    return 0;
  }
  return 1;
}

- (NSArray *)accessibilityElements
{
  if (!self.isNodeLoaded) {
    ASDisplayNodeFailAssert(@"Cannot access accessibilityElements since ASEditableTextNode is not loaded");
    return @[];
  }
  return @[self.textView];
}

- (id)accessibilityElementAtIndex:(NSInteger)index
{
  if (!self.isNodeLoaded) {
    ASDisplayNodeFailAssert(@"Cannot access accessibilityElementAtIndex: since ASEditableTextNode is not loaded");
    return nil;
  }
  return self.textView;
}

- (NSInteger)indexOfAccessibilityElement:(id)element
{
  return 0;
}
```

After that change let's get it in.